ReleaseNotes.md

```diff
@@ -7,6 +7,33 @@ samples distributed with the code.  The samples demonstrate the usage of compres
 public models and datasets for three different use cases: Image Classification, Object Detection,
 and Semantic Segmentation.
 
+## New in Release 2.0:
+- Added TensorFlow 2.4.2 support - NNCF can now be used to apply the compression algorithms to models originally trained in TensorFlow. 
+Limitations apply, and feature parity of the TF backend with the PyTorch will be improved in the upcoming releases.
+Current NNCF TF features include:
+    - Quantization (with HW-specific targeting aligned with PyTorch, custom layers not supported)
+    - Sparsity
+    - Filter pruning
+    - Batchnorm adaptation
+    - Algo mixing (quantization + pruning only)
+
```

Comment:
Perhaps 7-bit for weights could also be added.

Reply:
7-bit is a feature already introduce for PT in 1.7, so it is rather expected for TF to support it, can't specify it as a separate feature.

Reply:
Ok, I asked if we would like to list it in the list of TF features. It's more internal so it's up to you.